gdal/build.sh

```diff
@@ -1,7 +1,21 @@
 #!/bin/bash
+./configure --with-python --prefix=$PREFIX \
+--with-geos=$PREFIX/bin/geos-config \
+--with-static-proj4=$PREFIX \
+--with-hdf5=$PREFIX \
+--with-hdf4=$PREFIX \
+--with-netcdf=$PREFIX \
+--with-xerces=$PREFIX \
+--with-sqlite3=$PREFIX \
+--without-pam \
+--disable-rpath
```

Comment:
At least, I am pretty sure it isn't relocatable anymore. It is entirely possible that I am wrong about this. I won't be able to test this PR until tomorrow when I can get back to the office (I am a bit snowed in at the moment...)